### ISSUE TITLE
cluster_types/clusters initializers should run before devices

### ISSUE THREAD

**Issue by chamberk** [netbox-community/netbox-docker#365]
## Current Behavior

When using the startup_scripts initializers to create a device that references a cluster, it fails because the cluster does not exist yet. This is because the 140_cluster_types.py and 170_clusters.py are ordered after 130_devices.py. 

## Expected Behavior

The cluster_type and cluster should be created first. Then the device should be created and can reference a cluster. 

## Debug Information

The output of `docker-compose version`: `1.27.4`
The output of `docker version`: `19.03.13`
The output of `git rev-parse HEAD`: `a51ad368017dc7543855946ef3a031cfe5cba10e`
The command you used to start the project: `docker-compose up -d`

The output of `docker inspect netboxcommunity/netbox:latest --format "{{json .Config.Labels}}"`:

```json
{

}
```

The output of `docker-compose logs netbox`:

```text
Running the startup script /opt/netbox/startup_scripts/130_devices.py
Traceback (most recent call last):
  File "./manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.7/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.7/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.7/site-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.7/site-packages/django/core/management/base.py", line 364, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python3.7/site-packages/django/core/management/commands/shell.py", line 92, in handle
    exec(sys.stdin.read())
  File "<string>", line 1, in <module>
  File "/usr/local/lib/python3.7/runpy.py", line 280, in run_path
    run_name, mod_spec, pkg_name).copy()
  File "/usr/local/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "../startup_scripts/__main__.py", line 31, in <module>
    runpy.run_path(f.path)
  File "/usr/local/lib/python3.7/runpy.py", line 263, in run_path
    pkg_name=pkg_name, script_name=fname)
  File "/usr/local/lib/python3.7/runpy.py", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File "/usr/local/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/opt/netbox/startup_scripts/130_devices.py", line 43, in <module>
    params[assoc] = model.objects.get(**query)
  File "/usr/local/lib/python3.7/site-packages/django/db/models/manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/cacheops/query.py", line 390, in get
    return qs._no_monkey.get(qs, *args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/django/db/models/query.py", line 408, in get
    self.model._meta.object_name
virtualization.models.DoesNotExist: Cluster matching query does not exist.
```

**Comment by cimnine** [comment]
Thank you for bringing this to our attention. I haven't had time to investigate this thoroughly but there's not reason to doubt your observation anyway.

Feel free to open a PR to suggest an improved order.

**Comment by cimnine** [comment]
This issue should be resolved in the next version. It was resolved with #365.
